tests/kernel/update_kernel.pm

```diff
@@ -457,42 +504,50 @@ sub run {
         reboot_on_changes;
     }
 
+    my $repo = is_sle_micro('>=6.0') ? get_var('OS_TEST_REPOS') : get_var('KOTD_REPO');
+    my $incident_id = undef;
+
     add_extra_customer_repositories;
 
     if (get_var('KERNEL_VERSION')) {
-        downgrade_kernel(get_var('KERNEL_VERSION'));
+        my $kver = get_var('KERNEL_VERSION');
+
+        if ($repo) {
+            $self->add_update_repos($repo);
+            $self->enable_update_repos(0) unless get_var('DOWNGRADE_UPDATE');
```

Comment:
Variable name is little be confusing. Can we try something like `KERNEL_VERSION_IN_UPDATE' /  `KERNEL_VERSION_UPDATE' ?

Reply:
Renamed to `NO_DISABLE_REPOS`.